docs/examples/academy/28. Aggregating Cargo Movements Data.ipynb

```diff
@@ -382,152 +327,173 @@
+       "      <td>Medium-Sour</td>\n",
+       "      <td>3100037</td>\n",
        "    </tr>\n",
        "  </tbody>\n",
        "</table>\n",
        "</div>"
       ],
       "text/plain": [
        "    loading_week product_category  quantity\n",
-       "0             21      Light-Sweet  14457314\n",
-       "1             21      Medium-Sour   1557064\n",
-       "2             22      Light-Sweet  18779321\n",
-       "3             22      Medium-Sour    986121\n",
-       "4             23       Heavy-Sour   1584078\n",
-       "5             23      Light-Sweet  17238857\n",
-       "6             23      Medium-Sour   1959945\n",
-       "7             24       Heavy-Sour    483880\n",
-       "8             24       Light-Sour   1443458\n",
-       "9             24      Light-Sweet  19866926\n",
-       "10            25       Heavy-Sour    298671\n",
-       "11            25      Light-Sweet  24730561\n",
-       "12            25      Medium-Sour    357009\n",
-       "13            26       Light-Sour    815856\n",
-       "14            26      Light-Sweet  14701277\n",
-       "15            26      Medium-Sour   3447476\n",
-       "16            27      Light-Sweet  23007185\n",
-       "17            27      Medium-Sour   1275061\n",
-       "18            28      Light-Sweet   5623297"
+       "0              9      Light-Sweet   4898641\n",
+       "1              9      Medium-Sour   1279220\n",
+       "2             10       Heavy-Sour   1292891\n",
+       "3             10      Light-Sweet  23657246\n",
+       "4             10      Medium-Sour   1068981\n",
+       "5             11       Heavy-Sour   1168289\n",
+       "6             11      Light-Sweet  31123847\n",
+       "7             11      Medium-Sour   4418532\n",
+       "8             12       Heavy-Sour    291878\n",
+       "9             12      Light-Sweet  27299960\n",
+       "10            12      Medium-Sour   1083332\n",
+       "11            13      Light-Sweet  26522943\n",
+       "12            13      Medium-Sour    904399\n",
+       "13            14       Heavy-Sour    317250\n",
+       "14            14      Light-Sweet  16403216\n",
+       "15            14      Medium-Sour    775952\n",
+       "16            14     Medium-Sweet    474686\n",
+       "17            15       Heavy-Sour    876933\n",
+       "18            15      Light-Sweet  29190571\n",
+       "19            15      Medium-Sour   1077061\n",
+       "20            16      Light-Sweet  19109858\n",
+       "21            16      Medium-Sour   3100037"
       ]
      },
-     "execution_count": 10,
+     "execution_count": 13,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "quantity_by_category = cms.groupby(by = ['loading_week','product_category']).sum().reset_index()\n",
+    "quantity_by_category = cms.groupby(by = ['loading_week','product_category']).sum(numeric_only=True).reset_index()\n",
```

Comment:
This is a meaningful change in this file - it fixes an issue where the `.sum` method was not working after a Pandas upgrade.